zookeeper-server/src/main/java/org/apache/zookeeper/client/ZooKeeperSaslClient.java

```diff
@@ -289,8 +297,9 @@ public void respondToServer(byte[] serverToken, ClientCnxn cnxn) {
         if (saslClient.isComplete()) {
             // GSSAPI: server sends a final packet after authentication succeeds
             // or fails.
-            if ((serverToken == null) && (saslClient.getMechanismName().equals("GSSAPI")))
+            if ((serverToken == null) && (saslClient.getMechanismName().equals("GSSAPI"))) {
```

Comment:
A thought - would we like our checkstyle to flag "variable.equals(constant)" as less preferable than "constant.equals(variable)"?